lib/rs/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rift"
+description = "Rust Thrift library"
+version = "0.5.1"
+license = "Apache-2.0"
+authors = ["Allen George <allen.george@gmail.com>"]
+readme = "README.md"
+documentation = "https://docs.rs/rift/"
+exclude = ["Makefile*"]
+keywords = ["thrift"]
+
```

Comment:
Yep, fine. We also have a template on the bottom of [that page over here]((http://thrift.apache.org/docs/committers/HowToVersion)). The `name` should probably not be "Rift" and `documentation` should point to http://thrift.apache.org. The `version` tag needs to be the same as for all other client libraries.